src/LCT.PackageIdentifier/Program.cs

```diff
@@ -82,9 +82,17 @@ static async Task Main(string[] args)
             IBomCreator bomCreator = new BomCreator();
             bomCreator.JFrogService = GetJfrogService(appSettings);
             bomCreator.BomHelper = new BomHelper();
-            await bomCreator.GenerateBom(appSettings, new BomHelper(), new FileOperations());
 
-            Logger.Logger.Log(null, Level.Notice, $"End of Package Identifier execution : {DateTime.Now}\n", null);
+            //Validating JFrog Settings
+            if (await bomCreator.CheckJFrogConnection())
+            {
+                await bomCreator.GenerateBom(appSettings, new BomHelper(), new FileOperations());
+                Logger.Logger.Log(null, Level.Notice, $"End of Package Identifier execution : {DateTime.Now}\n", null);
+            }
+            else
+            {
+                Logger.Logger.Log(null, Level.Alert, $"JFrog Connection was not successfull!! \n", null);
```

Comment:
isn't this already being logged by the method  CheckJFrogConnection(), why again ?